monitoring/monitoring-overview.adoc

```diff
@@ -9,14 +9,17 @@ toc::[]
 [id="about-openshift-monitoring"]
 == About {product-title} monitoring
 
+[role="_abstract"]
 {product-title} includes a preconfigured, preinstalled, and self-updating monitoring stack that provides monitoring for core platform components. You also have the option to xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[enable monitoring for user-defined projects].
 
 A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations. {product-title} delivers monitoring best practices out of the box.
 
 A set of alerts are included by default that immediately notify cluster administrators about issues with a cluster. Default dashboards in the {product-title} web console include visual representations of cluster metrics to help you to quickly understand the state of your cluster. With the {product-title} web console, you can xref:../monitoring/managing-metrics.adoc#managing-metrics[view and manage metrics], xref:../monitoring/managing-alerts.adoc#managing-alerts[alerts], and xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[review monitoring dashboards]. 
 
-After installing {product-title} {product-version}, cluster administrators can optionally enable monitoring for user-defined projects. By using this feature, cluster administrators, developers, and other users can specify how services and pods are monitored in their own projects.
-As a cluster administrator, you can find answers to common problems such as user metrics unavailability and Prometheus consuming a lot of disk space in xref:../monitoring/troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues[troubleshooting monitoring issues].
+In the *Observe* section of {product-title} web console, you can access and manage monitoring features such as xref:../monitoring/managing-metrics.adoc#managing-metrics[metrics], xref:../monitoring/managing-alerts.adoc#managing-alerts[alerts], xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[monitoring dashboards], and xref:../monitoring/managing-metrics-targets.adoc#managing-metrics-targets[metrics targets].
+
+After installing {product-title} {product-version}, cluster administrators can optionally enable monitoring for user-defined projects. By using this feature, cluster administrators, developers, and other users can specify how services and pods are monitored in their own projects. 
```

Comment:
Does it make sense to use this continuously updating attribute, `{product-version}`, here? 
Features described by each release of the documentation are release-specific. Maybe we can remove this attribute?

Reply:
I removed the version attribute. You're right. It's not needed here.